mungegithub/mungers/publish_scripts/clientgo_publish.sh

```diff
@@ -43,6 +44,16 @@ if git diff --cached --exit-code &>/dev/null; then
     exit 0
 fi
 git commit -m "${MESSAGE}"
+
+godep restore
+godep save ./...
+git add --all
+if git diff --cached --exit-code &>/dev/null; then
+    echo "dependency has not changed!"
+else
+    git commit -m "update dependency, should only contain changes in k8s.io/apimachinery"
```

Comment:
"Pick up new dependencies"?

I don't recommend including references to specific packages when it looks like the above could have updated lots of stuff.